[WIP] init

hieki opened the pull request:
# What
TS と Webpack と Electron の環境は適当に用意したけど `npm start` すると electron がうまくトランスパイル後のコードを読んでくれないあたりまで作業した。
最低限動くコードにしてから merge する。

# みなさん
ご自由にこれの続きの作業をしてもらって構いませんよ :smiley:

hieki committed:
[WIP] init

TS と Webpack と Electron の環境は適当に用意したけど `npm start` すると electron がうまくトランスパイル後のコードを読んでくれないあたりまで作業した

Mic-U commented:
> electron がうまくトランスパイル後のコードを読んでくれない

webpack.config.js の target を指定すれば良さそう

- [Target](https://webpack.js.org/configuration/target/)
- [electronでReactを使うためのwebpackの設定 \- 貝塚かるたの徒然日記](http://karuta-kayituka.hatenablog.com/entry/2018/04/02/200207)

Mic-U committed:
target を指定

ウィンドウを起動するためにとりあえず electron-main にしたが、場合によっては electron-renderer の方が適切かもしれない
